e2e/apq-subgraphs/gateway.config.ts

```diff
@@ -0,0 +1,20 @@
+import { defineConfig } from '@graphql-hive/gateway';
+
+let fetchCnt = 0;
+export const gatewayConfig = defineConfig({
+  transportEntries: {
+    greetings: {
+      options: {
+        apq: true,
```

Comment:
Transport entries is marked as an advanced feature today. Do we want to hide this behind this setting ?

Perhaps we can have a top level option instead ?

Reply:
They are not advanced. It is a way to configure the transport.

Reply:
I don't think it's advanced, but that's what we are advertising in the documentation ^^'

And the way transport entries work is never actually explain. It's shortly explained in the Subscriptions page because it is needed to configure websocket/http callback subscription transports

Reply:
We should fix it then yes. I agree it is a bit low level but i think we should cover each feature individial rather than grouping them into one single transportEntries documentation. What do you think?